pretty.js

```diff
@@ -46,12 +53,18 @@ function _lpadzero (aTarget, aLength, aPadChar) {
 function withSpaces (value, eol) {
   var lines = value.split(/\r?\n/)
   for (var i = 1; i < lines.length; i++) {
-    lines[i] = '    ' + lines[i]
+    lines[i] = indent(4) + lines[i]
```

Comment:
I would actually prefer not to pass through this `indent()`  function, can you remove it?

Reply:
May I ask why?

IMHO, it's easier to spot the intention with the function rather than manually count spaces.

Reply:
Even this is not production code, `indent` is going to soon become the heaviest function in rendering anything, and generically not my style. If you prefer some text instead of `'    '`, you can use a top level variable: being dynamic here is not necessary.